bundles/org.openhab.binding.qbus/src/main/java/org/openhab/binding/qbus/internal/handler/QbusRolHandler.java

```diff
@@ -0,0 +1,344 @@
+        String snr = getSN();
+        if (command instanceof UpDownType) {
+            UpDownType upDown = (UpDownType) command;
+            if (upDown == DOWN) {
+                if (snr != null) {
+                    qRol.execute(0, snr);
+                } else {
+                    thingOffline("No serial number configured for  " + rolId);
+                }
+            } else {
+                if (snr != null) {
+                    qRol.execute(100, snr);
+                } else {
+                    thingOffline("No serial number configured for  " + rolId);
+                }
+            }
+        } else if (command instanceof IncreaseDecreaseType) {
+            IncreaseDecreaseType inc = (IncreaseDecreaseType) command;
+            int stepValue = ((Number) this.getConfig().get(CONFIG_STEP_VALUE)).intValue();
+            Integer currentValue = qRol.getState();
+            int newValue;
+            int sendValue;
+            if (currentValue != null) {
+                if (inc == IncreaseDecreaseType.INCREASE) {
+                    newValue = currentValue + stepValue;
+                    // round down to step multiple
+                    newValue = newValue - newValue % stepValue;
+                    sendValue = newValue > 100 ? 100 : newValue;
+                    if (snr != null) {
+                        qRol.execute(sendValue, snr);
+                    } else {
+                        thingOffline("No serial number configured for  " + rolId);
+                    }
+                } else {
+                    newValue = currentValue - stepValue;
+                    // round up to step multiple
+                    newValue = newValue + newValue % stepValue;
+                    sendValue = newValue > 100 ? 100 : newValue;
+                    if (snr != null) {
+                        qRol.execute(sendValue, snr);
+                    } else {
+                        thingOffline("No serial number configured for  " + rolId);
+                    }
+                }
+            }
+        } else if (command instanceof PercentType) {
+            PercentType p = (PercentType) command;
+            int pp = p.intValue();
+            if (p == PercentType.ZERO) {
+                if (snr != null) {
+                    qRol.execute(0, snr);
+                } else {
+                    thingOffline("No serial number configured for  " + rolId);
+                }
+            } else {
+                if (snr != null) {
+                    qRol.execute(pp, snr);
+                } else {
+                    thingOffline("No serial number configured for  " + rolId);
+                }
+            }
+        }
+    }
+
+    /**
+     * Executes the command for screen slats position
+     */
+    private void handleSlatsposCommand(QbusRol qRol, Command command) {
+        String snr = getSN();
+        if (command instanceof org.openhab.core.library.types.UpDownType) {
```

Comment:
You could add the package to the imports.
```suggestion
        if (command instanceof UpDownType) {
```